src/app/pages/send/send-crypto-asset-form/family/bitcoin/components/useBitcoinSetFees.ts

```diff
@@ -0,0 +1,68 @@
+import { useMemo } from 'react';
+
+import { createMoney } from '@shared/models/money.model';
+
+import { baseCurrencyAmountInQuote } from '@app/common/money/calculate-money';
+import { formatMoney, i18nFormatCurrency } from '@app/common/money/format-money';
+import { BtcSizeFeeEstimator } from '@app/common/transactions/bitcoin/fees/btc-size-fee-estimator';
+import { useCurrentTaprootAccountUninscribedUtxos } from '@app/query/bitcoin/balance/bitcoin-balances.query';
+import { BtcFeeType, btcTxTimeMap } from '@app/query/bitcoin/bitcoin-client';
+import { useBitcoinFeeRate } from '@app/query/bitcoin/fees/fee-estimates.hooks';
+import { useCryptoCurrencyMarketData } from '@app/query/common/market-data/market-data.hooks';
+
+export function useBitcoinSetFees() {
+  const uninscribedUtxos = useCurrentTaprootAccountUninscribedUtxos();
+
+  const btcMarketData = useCryptoCurrencyMarketData('BTC');
+  const { data: feeRate } = useBitcoinFeeRate();
+
+  const feesList = useMemo(() => {
+    function getFiatFeeValue(fee: number) {
+      return `~ ${i18nFormatCurrency(
+        baseCurrencyAmountInQuote(createMoney(Math.ceil(txVBytes * fee), 'BTC'), btcMarketData)
+      )}`;
+    }
+
+    if (!feeRate) return [];
+    const txSizer = new BtcSizeFeeEstimator();
+    const { txVBytes } = txSizer.calcTxSize({
+      input_count: uninscribedUtxos.length,
```

Comment:
This logic assumes that the transaction is going to contain all uninscribed taproot UTXOs, which is likely not the case.

For calculating inscription transfer fees, we'll use TR utxos (though for now, just `1`).
For calculating btc transfers, the number of inputs is dependent on the amount being transferred. 

This is where it gets tricky. It could be that for a transfer of _x_ the low fee estimate, might use 3 UTXOs, and that a high fee estimate uses 5.

For this case, I think we'll need to run the coinselection in `determineUtxosForSpend` 3 times, with varying fee rates, to determine the selections.